Gemfile.lock

```diff
@@ -60,74 +60,74 @@ GIT
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (7.1.2)
-      actionpack (= 7.1.2)
-      activesupport (= 7.1.2)
+    actioncable (7.1.3)
+      actionpack (= 7.1.3)
+      activesupport (= 7.1.3)
```

Comment:
this is updating a lot more than just `bundler`, what command did we run to generate this diff?

Reply:
I installed the latest bundler on my local machine then ran `bundle` and `bundle update`

Reply:
`bundle update` updates all available gems. Just `bundle` will run the current ones.

Reply:
I pushed a commit 74f4b4ad36dbf8600f9abe69c0e8cd04b18fe32f where I ran:
```
bundle update --bundler
bundle
```
so it only updates bundler, not any other gems